infoportal-common/src/kobo/IKoboMeta.ts

```diff
@@ -49,12 +49,14 @@ export enum KoboMetaStatus {
   Committed = 'Committed',
   Pending = 'Pending',
   Rejected = 'Rejected',
+  Blank = 'Blank'
```

Comment:
Should not be added here. It's normal that is `undefined`. All the key is to properly handle undefined on the filter system.